files/zh-cn/learn/css/building_blocks/selectors/selectors_tasks/index.md

```diff
@@ -49,7 +49,7 @@ Try updating the live code below to recreate the finished example:
 在本例中，尝试在不添加 HTML 的情况下进行以下更改。
 
 - 链接文本的样式：使链接为橘色，被访问后变为绿色，当被 hover 时，移除链接文本的下划线。
-- 让容器里的第一个元素的字体大小为:150%，并且让这个元素的第一行是红色的。
+- 让 `container` 类里的第一个元素的字体大小为:150%，并且让这个元素的第一行是红色的。
```

Comment:
Could you give the reason why using "`container` class" instead of "container", as the [en-US documation](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Selectors_Tasks#task_3) just using "container" which means the `div` element (not the class).

Reply:
change reverted. 

I use `container class` because the code below is using `container` as class name, 
I thought it was more clearly as it is an explicitly given name.

```
<div class="container">
  <p>Veggies es <a href="http://example.com">bonus vobis</a>, proinde vos postulo essum magis kohlrabi welsh onion daikon amaranth tatsoi tomatillo melon azuki bean garlic.</p>
```